### PR TITLE
fix(roles): clarify they're additive

### PR DESCRIPTION
**Pull request by zuluecho9**
liaison work: 

clarify that user mgmt roles are additive, and
Add a sentence about the new user record switcher and how it makes you reauthenticate to NR

### CODE REVIEW

**Comment by a-sassman** [comment]
Line 24-25 says: "If the desired user record has a manual login (not single sign-on), you'll be  records for non-SSO users." This appears to be an extra space and/or a type before "records for non-SSO users."

**Comment by zuluecho9** [comment]
> Line 24-25 says: "If the desired user record has a manual login (not single sign-on), you'll be records for non-SSO users." This appears to be an extra space and/or a type before "records for non-SSO users."

oh thanks, I think I left some draft stuff I was doing in there or something

File: src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx [hunk 1]
```diff
@@ -19,7 +19,9 @@ As a New Relic user, you may have more than one user record using the same email
 
 ## Switch between user records [#switch]
 
-When you log in to New Relic, you can go to the user menu to switch to other user records that have the same email address. 
+When you log in to New Relic, you can go to the [user menu](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) to switch to other user records that have the same email address. When you choose a user record from the menu, you'll be prompted to log in for that user record (either directly to New Relic or via your identity provider).
+
+ If the desired user record has a manual login (not single sign-on), you'll be  records for non-SSO users.
```
Comment (a-sassman) [inline src/content/docs/accounts/accounts-billing/account-setup/multiple-logins-found.mdx]: There appears to be an extra space and/or a type before "records for non-SSO users."

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 2]
```diff
@@ -27,7 +27,7 @@ Here we'll explain how New Relic users get access to specific capabilities and s
...
 
 * A user's **[user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type)**: For organizations on [our usage-based pricing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), your users' user type is a billing factor. The user type is what sets the maximum allowed capabilities a user can access. It overrides all role-related access. It's meant to be a fairly long-term setting based on someone's expected New Relic duties.
-* A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups).
+* A user's assigned **roles**: after a user's user type is decided, **roles** can be used to more precisely control a user's access. Roles are sets of **capabilities**, which are the granular New Relic abilities (for example, the ability to modify New Relic APM settings). Roles are assigned by applying them to a [user group](#groups).
```
Comment (a-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:30]: Role or roles? Is it possible for a user to have more than one role?

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 3]
```diff
@@ -94,9 +94,15 @@ To change a user's group, use the [**User management** UI](/docs/accounts/accoun
 
 ## Roles [#roles]
 
-**Roles** are groupings of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities), which are the granular things New Relic users can do (for example, the ability to manage data retention). We have several default roles, described below. And Pro and Enterprise edition organizations can create custom roles.
+**Roles** are groupings of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities), which are the granular things New Relic users can do (for example, the ability to manage data retention). We have several [default roles](#standard-roles), and Pro and Enterprise edition organizations can create their own custom roles.
```
Comment (a-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:97]:
```suggestion
We offer several [default roles](#standard-roles), but organizations with Pro or Enterprise editions can create their own custom roles.
```

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx [hunk 4]
```diff
@@ -94,9 +94,15 @@ To change a user's group, use the [**User management** UI](/docs/accounts/accoun
...
 
-To view roles and their capabilities, go to the [**Access management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks#where) and click **Roles**. Note that the roles UI doesn't show [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings).
+Important points about roles: 
+
+* Roles are additive: users with multiple roles assigned have the total of all the permissions granted by those roles. For example, if you're in a group that gives you the `All product admin` role in an account, and in another group that gives you a `Read only` role for the same account, you have both roles, and are not restricted by the `Read only` role.
```
Comment (a-sassman) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:101]:
```suggestion
* Roles are additive: users with multiple roles assigned have the total of all permissions granted by those roles. For example, if you're in a group that gives you the `All product admin` role in an account, and in another group that gives you a `Read only` role for the same account, you have both roles, and are not restricted by the `Read only` role. 
```

**Review by a-sassman — DISMISSED** [review]

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts.mdx:30]: yeah they can have multiple, hence why we have to clarify they're additive.

**Review by a-sassman — APPROVED** [review]

**Pull request merged**